src/widgets/forms.ts

```diff
@@ -161,21 +161,23 @@ field[ns.ui('Options').uri] = function (
   if (!cases) {
     box.appendChild(errorMessageBlock(dom, 'No cases to Options form. '))
   }
-  let values
+  let values: { [uri: string]: any }
```

Comment:
Why not be more specific with  `let values: { [uri: string]: boolean }`?

Reply:
Because `kb.findTypeURIs` on line 166 returns Statement objects. But yeah I guess that doesn't matter, maybe?

Reply:
OK, that works fine indeed, done.